index.js

```diff
@@ -42,7 +42,11 @@ module.exports = async function(options) {
       // This would be very simple with await, but for some reason
       // it gets a premature return value of undefined from spinUp. Shrug.
       function attempt() {
-        if (aposes[site._id] && (aposCreatedAt[site._id] < site.updatedAt)) {
+        if (aposes[site._id] === 'pending') {
+          setTimeout(attempt, 100);
```

Comment:
do we ever give up on `attempt` ?

Reply:
currently no, so if apos startup somehow got stuck we'd be stuck for that site, but that's also true for our single-site projects.